packages/ui/src/core/autocomplete/autocomplete.component.tsx

```diff
@@ -212,26 +235,36 @@ const Autocomplete = intrinsicComponent<AutocompleteProps, HTMLDivElement>(
             placeholder={placeholder}
             fullWidth={fullWidth}
             isEllipsis
-            iconEnd={() => (
-              <Styled.InputIconEndContainer>
-                {renderLabelIconEnd &&
-                  renderLabelIconEnd({
-                    isMultiple,
-                    option:
-                      !isMultiple && formattedValue && typeof formattedValue === 'string'
-                        ? getOptionById(formattedValue)
-                        : null,
-                  })}
-
-                {!hideArrow && (
-                  <ArrowTick
-                    {...(!disabled && !readOnly ? { onClick: handleOpenMenuClick } : {})}
-                    type={open ? 'top' : 'bottom'}
-                    IconProps={{ size: size === Size.Md ? 11 : 10 }}
-                  />
-                )}
-              </Styled.InputIconEndContainer>
-            )}
+            iconEnd={isMultiple ? undefined : renderInputEndIcons}
+            extraContent={
+              isMultiple ? (
+                <Styled.InputIconEndContainer>
+                  <>
+                    <Button
+                      size="sm"
+                      color="link-basic-primary"
+                      onClick={(e) => {
+                        handleSelectAllOptions();
+                        e.stopPropagation();
+                      }}
+                    >
+                      Select all
```

Comment:
Select all and Clear all need to be props with default value, so we send translated text using props, you can check how we do in tagField

Also we we will need props for callbacks after clicking onSelectAll and onClear all.